Enterprise-CMCS/macpro-mdct-mcr
# MLR Design Feedback

## Summary

### Description

Opening MLR links in a new tab and adding the corresponding ARIA labels.

### Related ticket

N/A

### How to test

Click on any link in the `Instructions` accordion of MLR Reporting, verify it takes you to a new tab.

### Important updates

N/A

### Author checklist
